src/H5Oginfo.c

```diff
@@ -68,6 +68,8 @@ const H5O_msg_class_t H5O_MSG_GINFO[1] = {{
 #define H5O_GINFO_STORE_PHASE_CHANGE   0x01
 #define H5O_GINFO_STORE_EST_ENTRY_INFO 0x02
 #define H5O_GINFO_ALL_FLAGS            (H5O_GINFO_STORE_PHASE_CHANGE | H5O_GINFO_STORE_EST_ENTRY_INFO)
+/* Check if access at this point of given size is valid */
+#define IS_BUFFER_OVERFLOW(ptr, size, buffer_end) (((ptr) + (size) - 1) > (buffer_end))
```

Comment:
Adding a macro for this is a good idea, but this should be located in a central location for general use, like H5private.h.

Reply:
Also, such a central macro would start with H5_